delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/model/BlazegraphViewRejection.scala

```diff
@@ -191,4 +198,19 @@ object BlazegraphViewRejection {
     case JsonLdRejection.InvalidJsonLdFormat(id, rdfError) => InvalidJsonLdFormat(id, rdfError)
     case JsonLdRejection.DecodingFailed(error)             => DecodingFailed(error)
   }
+
+  implicit private[plugins] val blazegraphViewRejectionEncoder: Encoder.AsObject[BlazegraphViewRejection] =
+    Encoder.AsObject.instance { r =>
+      val tpe = ClassUtils.simpleName(r)
+      val obj = JsonObject(keywords.tpe -> tpe.asJson, "reason" -> r.reason.asJson)
+      r match {
+        case WrappedOrganizationRejection(rejection) => rejection.asJsonObject
```

Comment:
you did not handle the case of `WrappedBlazegraphClientError`. You probably want something similar to: https://github.com/BlueBrain/nexus/blob/master/delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ElasticSearchViewRejection.scala#L207

Also you didn't handle `InvalidJsonLdFormat`.